README.md

```diff
@@ -5,9 +5,9 @@
 
 ![winget install wingetcreate](.github/images/WingetInstall.gif)
 
-If you are new to the Windows Package Manager, you might want to [Explore the Windows Package Manager tool](https://docs.microsoft.com/learn/modules/explore-windows-package-manager-tool/?WT.mc_id=AZ-MVP-5004737). The client has access to packages from two default sources. The first is "msstore" the Microsoft Store (free Apps rated "e" for everyone). The second is "winget" the [WinGet community repository](https://github.com/microsoft/winget-pkgs).
+If you are new to the Windows Package Manager, you might want to [Explore the Windows Package Manager tool](https://docs.microsoft.com/learn/modules/explore-windows-package-manager-tool/?WT.mc_id=AZ-MVP-5004737). The client has access to packages from two default sources. The first is "msstore" the Microsoft Store (free Apps rated "e" for everyone). The second is "Winget" the [WinGet community repository](https://github.com/microsoft/winget-pkgs).
 
-> **Note**: Group policy may be configured and modify configured sources. Run `winget --info` to see any configured policies.
+> **Note**: Group policy may be configured and configured sources. Run `winget --info` to see any configured policies.
```

Comment:
This sentence doesn't make sense with your change. The original has proper grammar

Reply:
Then why did Grammarly suggest that?